From 4ba28a31b04e1e1df20fb063e4c2c9284c132084 Mon Sep 17 00:00:00 2001
From: jxlwqq <jxlwqq@gmail.com>
Date: Thu, 9 Dec 2021 02:16:08 +0800
Subject: Sample Jaeger version 1.23 -> 1.29 (#36433)

The docker.io/jaegertracing/all-in-one image supports multi-platform build since 1.24.

https://registry.hub.docker.com/layers/jaegertracing/all-in-one/1.24/images/sha256-5d86c2944deaf08f418d06939f5043791694b4fba3a8b258c514d0a06420e0af
---
 samples/addons/jaeger.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/samples/addons/jaeger.yaml b/samples/addons/jaeger.yaml
index d97c076414..4803f5a930 100644
--- a/samples/addons/jaeger.yaml
+++ b/samples/addons/jaeger.yaml
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
         - name: jaeger
-          image: "docker.io/jaegertracing/all-in-one:1.23"
+          image: "docker.io/jaegertracing/all-in-one:1.29"
           env:
             - name: BADGER_EPHEMERAL
               value: "false"
-- 
2.35.3

